test/unit/core.js

```diff
@@ -489,8 +493,56 @@ QUnit.test( "isFunction", function( assert ) {
 	callme( function() {
 		callme( function() {} );
 	} );
+
+	// Functions from other windows should be matched
+	Globals.register( "iframeDone" );
```

Comment:
I guess, but it seems to fit with the rest of these assertions pretty well. Would anyone else like to weigh in?